src/NSubstitute/NSubstitute.csproj

```diff
@@ -21,11 +21,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Castle.Core" Version="5.0.0-*" />
+    <PackageReference Include="Castle.Core" Version="5.1.1-*" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetIsNet5OrNewer)' != 'true'">
-    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.3.0-*" />
```

Comment:
I would not bump the lower limit of `System.Threading.Tasks.Extensions`. It's not required by NSub unlike the `Castle.Core` library update that fixes an issue.
We don't limit the upper boundary so users can update it on their own. I don't want to force that. NSub is a library not framework.

Reply:
Good point, I reverted this change. 👍